docs/fix-links/default.nix

```diff
@@ -26,8 +26,8 @@ runCommand (src.name or (builtins.baseNameOf src))
 
     pandoc \
         --output $out \
-        --from markdown \
-        --to markdown-raw_attribute \
+        --from gfm \
+        --to gfm \
```

Comment:
Maybe it'd be best to keep `--from gfm` but have `--to markdown-raw_attribute+alerts`?

This probably better fits what we're actually trying to target with mdbook's markdown support :thinking: